app/services/send_chase_email_to_referee_and_candidate.rb

```diff
@@ -1,7 +1,10 @@
 class SendChaseEmailToRefereeAndCandidate
   def self.call(application_form:, reference:)
     RefereeMailer.reference_request_chaser_email(application_form, reference).deliver
+    ChaserSent.create!(chased: reference, chaser_type: :reference_request)
+
     CandidateMailer.reference_chaser_email(application_form, reference).deliver
+    ChaserSent.create!(chased: application_form.candidate, chaser_type: :reference_request)
```

Comment:
I don't think we need this one, we're not using it right?

Reply:
Yea you're right 👍